docs/oracle-demo.md

```diff
@@ -0,0 +1,92 @@
+**Oracle agents** are agents that have permission to update or validate updates to state variables in a smart contract and whose goal is to accurately estimate or predict some real world quantity or quantities.
+
+This demonstration shows how to set up a simple oracle agent who deploys an oracle contract and updates the contract with a token price fetched from a public API.
+
+## Preparation instructions
+ 
+### Dependencies
+
+Follow the <a href="../quickstart/#preliminaries">Preliminaries</a> and <a href="../quickstart/#installation">Installation</a> sections from the AEA quick start.
+
+## Demo
+
+### Create the simple oracle AEA
+
+Fetch the AEA that will deploy and update the oracle contract.
+
+``` bash
+aea fetch fetchai/coin_price_oracle:0.1.0
+cd coin_price_oracle
+aea install
+```
+
+<details><summary>Alternatively, create from scratch.</summary>
+<p>
+
+Create the AEA that will deploy the contract.
+
+``` bash
+aea create coin_price_oracle
+cd coin_price_oracle_client
+aea add connection fetchai/http_client:latest
+aea add connection fetchai/ledger:latest
+aea add connection fetchai/p2p_libp2p:latest
+aea add --local skill fetchai/coin_price:0.1.0
+aea add --local skill fetchai/simple_oracle:0.1.0
```

Comment:
please drop local; when the docs are released this will match what is on registry